src/api.ts

```diff
@@ -46,6 +46,8 @@ const fromLocalHost: express.RequestHandler = (req, res, next) => {
 export const apiFactory = (serviceId: number) => {
 	const api = express.Router();
 
+	const workerMap: { [key: string]: string } = {};
```

Comment:
```suggestion
	const workerMap: _.Dictionary<string> = {};
```

Reply:
You have `lodash` in your deps anyway, so makes sense to use it's `Dictionary<>` type 👍